TicketBooking/UserInterface/ManagerInterface.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TicketBooking.Class;
+using TicketBooking.Flight.FlightModel;
+using TicketBooking.Flight.FlightRepository;
+using TicketBooking.User.Passenger.Bookings;
+using TicketBooking.User.Passenger.PassengerModel;
+using TicketBooking.User.UserRepository;
+
+namespace TicketBooking.UserInterface
+{
+    public class ManagerInterface
+    {
+        FlightRepository _flights = new FlightRepository();
```

Comment:
Can we have it `private readonly FlightRepository _flights`
same for bookings.

-- please give a better names 
`_flights` => `_flightsRepository`
`_bookings` => `_bookingsList` or something better.